api/tests/mocha/services/generate-xform.spec.js

```diff
@@ -334,8 +357,135 @@ describe('generate-xform service', () => {
       });
     });
 
+    it('should not update doc if attachment is not xml/html', async () => {
+      const newForm = '<html><title>Hello</title></html>';
+      const newModel = '<instance><multimedia/></instance>';
+      
+      sinon.stub(db.medic, 'getAttachment').resolves({ _attachments: {}});
+
+      sinon.stub(db.medic, 'get').resolves({ _attachments: {
+        'xform.xml': {stub: true},
+        'form.html': { stub: true },
+        'model.xml': { stub: true }
+      } });
+      sinon.stub(service, 'generate').resolves({ form: newForm, model: newModel });
+      sinon.stub(db.medic, 'put');
+
+      await service.update('form:exists');
+      expect(db.medic.put.callCount).to.equal(1);
```

Comment:
The test title says that the form should not be updated if the attachment is not xml, but then this assertion contradicts the test name, with the form actually getting updated. 
I'm confused.